semantics.json

```diff
@@ -182,6 +182,14 @@
     "importance": "low",
     "description": "These options will let you control how the task behaves.",
     "fields": [
+      {
```

Comment:
Could you please put this field after the `ignorePunctuation`? I try to keep these options in order of potential use, and using a different separator than a space is probably not the most likely one :-)